site-builder/src/site/config.rs

```diff
@@ -21,12 +21,16 @@ pub struct WSResources {
     /// The attributes used inside the Display object.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub metadata: Option<Metadata>,
+    /// The name of the site.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub name: Option<String>,
 }
 
 impl WSResources {
     /// Reads and parses the `ws-resources.json` file into a `WSResources` struct.
     pub fn read<P: AsRef<Path>>(path: P) -> Result<WSResources> {
         // Load the JSON contents to a string.
+        tracing::info!("Reading Walrus site resources: {}", path.as_ref().display());
```

Comment:
Nit: in logs, prefer using record fields and all-lowercase.
```suggestion
        tracing::info!(file=%path.as_ref().display(), "reading Walrus site resources");
```